config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  namespace :v1, defaults: { format: 'json' } do
+    #get 'things', to: 'things#index'
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.